testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -138,23 +138,30 @@ impl EmitJobRequest {
         instances: Vec<Instance>,
         global_emit_job_request: &Option<EmitJobRequest>,
         gas_price: u64,
+        invalid_tx: u64,
```

Comment:
At some point (depending on how much more complicated this ends up becoming) it might be worth while to refactor this to use a Builder pattern so that things like `invalid_tx` which are optional, don't have to be specified at every call site